Makefile.am

```diff
@@ -62,11 +62,6 @@ substitute_libexecdir = sed \
 	-e 's|@libexecdir[@]|$(libexecdir)|g' \
 	$(NULL)
 
-# systemd service units
-#
-# Defined as systemdsystemunitdir in ../configure.ac.
-dist_systemdsystemunit_DATA = 
```

Comment:
This  was only just moved to `Makefile.am` two patches ago!

Reply:
`git rebase` 😿😿😿😿